include/boost/process/v2/ext/impl/exe.ipp

```diff
@@ -159,12 +159,12 @@ filesystem::path exe(boost::process::v2::pid_type pid, boost::system::error_code
     std::size_t len = 0;
     if (sysctl(mib, 4, nullptr, &len, nullptr, 0) == 0) 
     {
-        std::string strbuff;
+        std::vector<char> strbuff;
```

Comment:
Why not string? With vector .data() is correct in C++11.

Reply:
Are you actually telling me to do something or merely pointing out why I did that?

Reply:
The string is getting modified. .data() is const in c++11.